WordPress/src/main/java/org/wordpress/android/ui/notifications/NotificationsDetailActivity.java

```diff
@@ -159,12 +164,20 @@ public void showPostActivity(long siteId, long postId) {
         ReaderActivityLauncher.showReaderPostDetail(this, siteId, postId);
     }
 
-    public void showStatsActivityForSite(int localTableSiteId) {
+    public void showStatsActivityForSite(int localTableSiteId, NoteBlockRangeType rangeType) {
         if (isFinishing()) return;
 
-        Intent intent = new Intent(this, StatsActivity.class);
+        Intent intent;
+        if (rangeType == NoteBlockRangeType.FOLLOW) {
+            intent = new Intent(this, StatsViewAllActivity.class);
+            intent.putExtra(StatsAbstractFragment.ARGS_VIEW_TYPE, StatsViewType.FOLLOWERS.ordinal());
```

Comment:
Yeah, I thought that was a bit weird when I saw how it worked. Filed https://github.com/wordpress-mobile/WordPress-Android/issues/2469